<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaceTeamMonitor</class>
 <widget class="QMainWindow" name="RaceTeamMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1468</width>
    <height>1065</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RaceTeamMonitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Ãœberblick</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label3_7">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Rennbeginn:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QDateTimeEdit" name="racestart_datetimeedit">
           <property name="minimumSize">
            <size>
             <width>190</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>3456364</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>10</hour>
             <minute>25</minute>
             <second>0</second>
             <year>2016</year>
             <month>7</month>
             <day>21</day>
            </datetime>
           </property>
           <property name="displayFormat">
            <string>dd MMM yyyy HH:mm:ss</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>208</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableWidget" name="race_tablewidget">
           <property name="minimumSize">
            <size>
             <width>429</width>
             <height>596</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Start</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ende</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Countdown</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Status</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fahrzeit</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>291</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Restzeit:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLCDNumber" name="lcd_race">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>291</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>291</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>261</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Uhrzeit:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QLCDNumber" name="lcd_clock">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>291</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>291</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>911</width>
        <height>351</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Wechselzone</string>
      </property>
      <widget class="QLabel" name="change_label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>901</width>
         <height>51</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>901</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>51</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>22</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>XXX</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="change_groupbox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>230</y>
         <width>581</width>
         <height>95</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>581</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift SemiLight</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Wechsel erfolgte vor ca.</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="change_5min_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>5 min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="change_2min_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>2 min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="change_1min_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>1 min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="change_30sek_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>30 sek</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="change_10sek_pushbutton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>10 sek</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Auswertung</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="plot_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiLight</family>
           <pointsize>13</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="stats_tablewidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>221</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift SemiLight</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <row>
          <property name="text">
           <string>Anzahl Runden</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>gesamte Fahrzeit</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Schnitt Rundenzeit</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Schnitt Speed</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Andi</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Chris</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1468</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEinstellungen">
    <property name="title">
     <string>Einstellungen</string>
    </property>
   </widget>
   <addaction name="menuEinstellungen"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../TrackMyLaps/lapcounter.qrc"/>
 </resources>
 <connections/>
</ui>
